reproducible/ubuntu/mknodeimage.sh

```diff
@@ -1,19 +1,29 @@
 #!/bin/bash
 set -ex
 
+VERSION=$1
+
 ln -s /usr/bin/python2.7 /usr/bin/python
```

Comment:
Some comments in this file would help to explain how it works, what arguments it takes and what it does.